README.md

```diff
@@ -86,7 +89,7 @@ The dcrdata repository is a collection of golang packages and apps for [Decred](
 
 ## Requirements
 
-- [Go](http://golang.org) 1.10.x or 1.11.x.
+- [Go](http://go.org) 1.10.x or 1.11.x.
```

Comment:
well, the website is golang.org

Reply:
Oops.  I did a replace all.  Let me fix.